domino_data/_feature_store/run_featurestore_sync.py

```diff
@@ -0,0 +1,12 @@
+"""Containing script to sync feature store from feast to domino"""
+import sys
+
+from ..logging import logger
+from .featurestore_sync import feature_store_sync
+
+if __name__ == "__main__":
+    try:
+        feature_store_sync()
```

Comment:
I was expecting this script entry point to take multiple arguments (with sane defaults) like commit sha, repo path, feature store id..?